.github/ISSUE_TEMPLATE/tc2_bug_report.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Team Comtress bug report
+about: A bug that occurs in Team Comtress
+title:
+labels: bug, tc2, needs triage
+assignees:
+
+---
+
+### Description
+<!--
+A clear and concise description of what the bug is.
+-->
+
+### Checklist
+<!-- You do not have to answer "yes" to all of these to open an issue. -->
+- [ ] None of the *open or closed* issues document this problem.
+- [ ] This is a bug, not a new idea.
+- [ ] This is obviously a bug, not something that people commonly disagree on.
+- [ ] This bug is in the code of the game, not in the textures, sounds, or other resources.
+- [ ] This issue occurs on the lastest vanilla TF2 update.
+- [ ] This issue is listed on the [Source 1 official bugtracker](https://github.com/ValveSoftware/Source-1-Games/issues).
+- [ ] This issue occurs with DirectX level 100.
+- [ ] This issue occurs on the latest Team Comtress release, or the `master` branch.
+- [ ] This issue occurs when using no custom content.
+- [ ] This issue occurs when using `start_tf2.bat` (or `start_server.bat`) with no custom launch options.
+- [ ] This issue occurs when using a server started with the `map` command from the game.
+- [ ] This issue occurs when using a dedicated server.
+- [ ] This bug is in the internal procedures or tooling (e.g. the build process).
+
+### Reproduction
+<!--
+The steps to recreate the bug.
+Bugs that cannot be recreated are extremely unlikely to receive a fix.
+-->
+1.
+
+### Screenshots
+<!-- Add screenshots to help explain your problem. -->
+
+### Setup Details
+- *Server or client:* <!-- Did you observe this issue from the game client, or by running a dedicated server? -->
+- *Operating system:* <!-- Specify a version e.g. Windows 10, Mac OS X Catalina, or `uname -rv` output. -->
+- *Team Comtress version:* <!-- Release version number or commit id -->
+- *Custom content:* <!-- Any 3rd-party content, including config files -->
+- *Launch options:* <!-- If not using start_tf2.bat, specify a list of options -->
+<!-- For graphics-related issues: -->
+- *Graphics driver:* <!-- Specify a version. This is especially important on Linux. -->
+- *DirectX level:* <!-- This will be different if you set dxlevel in vanilla TF2. -->
+
+### Performance Details
+<!-- Fill this out for performance-related issues. -->
+- *GPU model:* <!-- Example: NVIDIA GTX 2070 Super -->
+- *CPU model:* <!-- Example: AMD RYZEN 3600X -->
+- *RAM amount:* <!-- Example: 32GB -->
+- *TF2 installed on:* <!-- HDD or SSD. Leave empty if not yet tested with vanilla TF2. -->
+- *TC2 installed on:* <!-- HDD or SSD -->
```

Comment:
I recommend adding this line afterward, as well as in all performance sections:
`- *TC2 installation method used:* <!-- Basic or Advanced -->`
This will be exceedingly relevant once the readme is refactored to mention the [Advanced Install Instructions](https://github.com/mastercomfig/team-comtress-2/wiki/Advanced-Installation-Instructions) from the wiki, as the basic instructions do not provide a 'clean' environment, and might retain some settings from previous live TF2 installs (ex. using a graphics config)

Reply:
Ok, tho i don't quite like the phrasing. We can fix it in the future when the README and wiki changes get merged